Makefile

```diff
@@ -6,22 +6,12 @@ virtualenv:
 	[ -z $$VIRTUAL_ENV ] && [ ! -d venv ] && python3 -m venv venv || true
 
 .PHONY: freeze-requirements
-freeze-requirements:
-	rm -rf venv-freeze
-	python3 -m venv venv-freeze
-	$$(pwd)/venv-freeze/bin/pip install -r requirements-app.txt
-	echo '# This file is autogenerated. Do not edit it manually.' > requirements.txt
-	cat requirements-app.txt >> requirements.txt
-	echo '' >> requirements.txt
-	$$(pwd)/venv-freeze/bin/pip freeze -r <(sed '/^--/d' requirements-app.txt) | sed -n '/The following requirements were added by pip freeze/,$$p' >> requirements.txt
-	rm -rf venv-freeze
+freeze-requirements: virtualenv requirements-dev requirements-app.txt
+	${VIRTUALENV_ROOT}/bin/python -m dmutils.repoutils.freeze_requirements requirements-app.txt
 
-.PHONY: test-requirements
-test-requirements:
-	@diff requirements-app.txt requirements.txt | grep '<' \
-	    && { echo "requirements.txt doesn't match requirements-app.txt."; \
-	         echo "Run 'make freeze-requirements' to update."; exit 1; } \
-	    || { echo "requirements.txt is up to date"; exit 0; }
```

Comment:
Why are we getting rid of `test-requirements` here?

Reply:
Replaced this. Accidentally removed.

Reply:
Sorry, actually, I removed it because it's duplicated in that file.

Reply:
Hard to keep track 😬

Reply:
Cool 😸Works locally for me